docs/terraform/terraform.md

```diff
@@ -5,55 +5,53 @@
 - [Terraform](https://www.terraform.io/downloads.html) >= 0.13.x
 - [Go](https://golang.org/doc/install) 1.16 to build the provider plugin
 
-## Install The Provider
+## Install the Provider
 
-### Option 1: Download and Install The Provider From [Releases](https://github.com/harvester/terraform-provider-harvester/releases)
-```bash
+### **Option 1:** Download and install the provider from [releases](https://github.com/harvester/terraform-provider-harvester/releases) using the following command:
```

Comment:
Could we move the following command part to a separate line. It looks a bit awkward right now having a heading that has a colon near the start and ending with one